content/influxdb/v2.0/reference/flux/stdlib/experimental/usage/from.md

```diff
@@ -0,0 +1,148 @@
+---
+title: usage.from() function
+description: >
+  The `usage.from()` function returns usage data from an **InfluxDB Cloud** organization.
+menu:
+  influxdb_2_0_ref:
+    name: usage.from
+    parent: usage-exp
+weight: 401
+influxdb/v2.0/tags: [functions, cloud, usage]
+---
+
+The `usage.from()` function returns usage data from an **InfluxDB Cloud** organization.
+
+```js
+import "experimental/usage"
+
+usage.from(
+  start: -30d,
+  stop: now(),
+  host: "",
+  orgID: "",
+  token: "",
+  raw: false
+)
+```
+
+{{< expand-wrapper >}}
+{{% expand "View usage data schema" %}}
+- **storage_usage_bucket_bytes** measurement
+  - **gauge** field
+  - **bucket_id** tag
+  - **org_id** tag
+- **http_request** measurement
+  - **req_bytes** field
+  - **resp_bytes** field
+  - **org_id** tag
+  - **endpoint** tag
+  - **status** tag
+- **query_count** measurement
+  - **req_bytes** field
+  - **endpoint** tag
+  - **orgID** tag
+  - **status** tag
+
+<!--
+- **queryd_billing** measurement
+- **storage_usage_bucket_cardinality** measurement
+- **events** measurement
+-->
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+
+## Parameters
+
+### start
+({{< req >}})
+Earliest time to include in results.
```

Comment:
Shall we link to/describe valid duration types or safe to assume folks know?...global

Reply:
I think it's safe to assume users know.